tests/libs/util/native_helper.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "catch_wrapper.hpp"
+#include "ebpf_execution_type.h"
+#include "native_helper.hpp"
+
+#include <rpc.h>
+
+static std::string
+_guid_to_string(_In_ const GUID* guid)
+{
+    char guid_string[37] = {0};
+    sprintf_s(
+        guid_string,
+        sizeof(guid_string) / sizeof(guid_string[0]),
+        "%08x-%04x-%04x-%02x%02x-%02x%02x%02x%02x%02x%02x",
+        guid->Data1,
+        guid->Data2,
+        guid->Data3,
+        guid->Data4[0],
+        guid->Data4[1],
+        guid->Data4[2],
+        guid->Data4[3],
+        guid->Data4[4],
+        guid->Data4[5],
+        guid->Data4[6],
+        guid->Data4[7]);
+
+    return std::string(guid_string);
+}
+
+#pragma warning(push)
+#pragma warning(disable : 6101) // Returning uninitialized memory '*unicode_string'
+_Success_(return == 0) int32_t string_to_wide_string(_In_z_ const char* input, _Outptr_ wchar_t** output)
```

Comment:
I don't see anywhere that calls this function, is it used?  Or is it dead code that should be deleted?

Reply:
Yes I missed to clean this up (had added earlier, but then did not use). Cleaned up now.